app/models/katello/package_category.rb

```diff
@@ -0,0 +1,6 @@
+module Katello
+  class PackageCategory < Katello::Model
+    include Concerns::PulpDatabaseUnit
+    CONTENT_TYPE = "package_category".freeze
+  end
+end
```

Comment:
These are model classes, but do not support indexing.  I wouldn't have them inherit from Katello::Model, nor include the Concerns::PulpDatabaseUnit concern (since they don't exist in the db)